galaxy_ng/app/migrations/0042_legacyroletag_legacyrole_tags.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 4.2.6 on 2023-10-25 21:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("galaxy", "0041_alter_containerregistryremote_remote_ptr"),
```

Comment:
`0042_namespace_created_namespace_updated` ?

(there's a new migration if you rebase .. and this bit seems to produce an infinite loop when starting)

Reply:
Good catch :+1:, rebased.